src/main/resources/avro/bdg.avdl

```diff
@@ -780,39 +780,189 @@ enum Allele {
+  union { null, float } expectedAlleleDosage = null; // or expectedAlleleCount? see Variant.alleleCount
+
+  // GL Number=G Type=Float
+  /**
+   Genotype likelihoods comprised of floating point log10-scaled likelihoods for all possible
+   genotypes given [the set of alleles defined in the REF and ALT fields]. VCF genotype field
+   reserved key GL, Number=G.
+   */
+  array<float> likelihoods = [];
+
+  // GLE Number=1 Type=String
+  // Genotype likelihoods of heterogeneous ploidy, used in presence of uncertain copy number.
+  // Reserved genotype field key in VCF v4.2, dropped in VCF v4.3
+
+  // GP Number=G Type=Float
+  /**
+   Genotype posterior probabilities in the range 0.0 to 1.0 using the same ordering as the
+   genotype likelihood field; one use can be to store imputed genotype probabilities. VCF genotype
+   field reserved key GP, Number=G.
+   */
+  array<float> posteriorProbabilities = [];
+
+  // GQ Number=1 Type=Integer
+  /**
    Conditional genotype quality, encoded as a phred quality −10log10 p(genotype call is wrong
    conditioned on the site being variant). VCF genotype field reserved key GQ.
    */
-  union { null, int } genotypeQuality = null;
+  union { null, int } quality = null;
 
+  // HQ Number=2 Type=Integer
   /**
-   Haplotype quality, encoded as two phred qualities. VCF genotype field reserved key HQ.
+   Haplotype quality, encoded as two phred qualities. VCF genotype field reserved key HQ, Number=2.
    */
   array<int> haplotypeQuality = [];
 
+  // MQ Number=1 Type=Integer
+  /**
+   Root Mean Square (RMS) of the mapping quality of reads for this genotype. VCF genotype field
+   reserved key MQ.
+   */
+  union { null, int } rmsMappingQuality = null;
+
+  // PL Number=G Type=Integer
+  /**
+   Phred-scaled genotype likelihoods rounded to the closest integer, and otherwise defined precisely
+   as the genotype likelihoods field. VCF genotype field reserved key PL, Number=G.
+   */
+  array<int> phredScaledLikelihoods = [];
+
+  // PQ Number=1 Type=Integer
   /**
    Phred-scaled probability that alleles are ordered incorrectly, against all other members in
    the same phase set. VCF genotype field reserved key PQ.
    */
   union { null, int } phaseSetQuality = null;
 
-  /**
-   Root Mean Square (RMS) of the mapping quality of reads for this genotype. VCF genotype field
-   reserved key MQ.
-   */
-  union { null, int } rmsMappingQuality = null;
 
+  // SB is not a reserved key in VCF v4.2 or v4.3
   /**
    Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias.
    If set, the array should contain the number of reads that support the reference allele on the
    forward strand, the reference allele on the reverse strand, the alternate allele on the forward
    strand, and the alternate allele on the reverse strand, in that order. VCF genotype field key SB.
    */
-  array<int> strandBias = [];
+  //array<int> strandBias = [];  Genotype.strandBiasComponents previously
```

Comment:
I'll take a look. I am -1 on dropping `strandBias`, and I prefer the old name (`strandBiasComponents`). You can't compute the strand bias components without going back to the reads, and the strand bias info is pretty important for variant filtration.